submissions/cynthiaNwakaeme/domManipulation/signUp.js

```diff
@@ -0,0 +1,68 @@
+let submitForm = document.getElementById("validateForm"),
+    firstName = document.getElementById("firstName"),
+    lastName = document.getElementById("lastName"),
+    phoneNumber = document.getElementById("phoneNumber"),
+    email = document.getElementById("email"),
+    password = document.getElementById("password"),
+    confirmPassword = document.getElementById("confirmPassword");
+
+
+validate = (field, event, regEx, result, output) => {
```

Comment:
I believe this is a function declaration, Please make use of the `var` or `let` keyword. Same goes for others where you declare a function as a variable name

Reply:
Hello @adaugochi is it mandatory inside var or let for ES6 function (arrow function).

Reply:
It might work fine. But, I don't think it is a good coding practice

Reply:
Hello @adaugochi . I really do appreciate the fact you took your time to explain this to me.
I have effected the changes ma.


Kindly approve my PR.
Thank You so much.